### DISCUSSION TITLE
License of the webpage design and code?

### DISCUSSION THREAD

**Discussion by deadrat-in** [genicsblog/genicsblog.com#136] [Q&A] [answered]
Kindly let me know if your webpage design and code (everything except the content in _posts) is available under any open source license.

**Comment by gouravkhunger** [comment, marked as answer]
Hey @Rat-S!

The theme content for the publication is managed under the [`theme-files`](https://github.com/genicsblog/theme-files) repo. We use a custom build process to [merge everything before generating](https://github.com/genicsblog/genicsblog.com/blob/main/.github/workflows/publish-site.yml#L20) the site.

Yes, the theme is open source under the [MIT License](https://github.com/genicsblog/theme-files/blob/main/LICENSE.md) but with custom attribution terms. If you wish to use the publication theme under open source terms, we require you to preserve the attribution to the developer (Gourav Khunger) and contributors in the [site footer](https://github.com/genicsblog/theme-files/blob/main/_includes/footer.html#L7). Referring to this line:

<img width="481" alt="image" src="https://user-images.githubusercontent.com/46792249/165120984-de34e87b-8b7b-4127-8053-a853c91f2df1.png">

You can modify the sentence but the links should be preserved :)

If you want to use it for commercial purposes, I am going to release a refined paid version of the theme specifically for public use (currently it has a lot of hardcoded values) within few days, so if you are fine with waiting, it could better help your use case!

**Reply by deadrat-in** [reply]
Got it, thanks

**Reply by gouravkhunger** [reply]
Great! If this helped, do mark it as resolved :)
Thanks for contacting!
